analyzer/queries/queries.go

```diff
@@ -625,6 +608,7 @@ var (
       LEFT JOIN chain.address_preimages AS account_preimage ON
         account_preimage.address = balance_analysis.account_address
       WHERE
+        evm_tokens.token_type IS NOT NULL AND
```

Comment:
before, `token_type` was guaranteed to be `NOT NULL`. Now, a type of `NULL` means "this is not a token for sure yet, but it's a candidate"